transport/session.go

```diff
@@ -324,7 +324,7 @@ func (s *session) SetWaitTime(waitTime time.Duration) {
 	s.wait = waitTime
 }
 
-// set task pool
+// Deprecated: set task pool
```

Comment:
the func inner implement can be removed, just leave a empty func for compatibility

Reply:
an empty func will make users lose original logic.